scripts/openapitools.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
where did this file come from?. 
I think it should be also deleted, there is already a openapitools file in the root folder